src/modules/components/vote/voteProposalDataListItem/voteProposalDataListItemStructure/voteProposalDataListItemStructure.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import { DataList, Tag, type IDataListItemProps } from '../../../../../core';
+import { voteIndicatorToTagVariant } from '../../voteDataListItem/voteDataListItemStructure/voteDataListItemStructure';
+
+export interface IVoteProposalDataListItemStructureProps extends IDataListItemProps {
+    /**
+     * The ID of proposal.
+     */
+    id: string;
+    /**
+     * The account details of the voter.
+     */
+    title: string;
```

Comment:
I would call these two properties `proposalId` & `proposalTitle` to clarify that they are related to the proposal entity and not the vote one
```suggestion
    proposalId: string;
    /**
     * The title of the proposal the user voted on.
     */
    proposalTitle: string;
```